lesson3.js

```diff
@@ -0,0 +1,15 @@
+const ul = document.getElementById('js-list');
+for(let i=1; i<=2; i++) {
```

Comment:
2 is magic number.
Can you explain what this 2 means and what it means?

hint
This is the number of elements in the array, right?

Reply:
Thanks for the tip to solve this problem.
2 is a number in the array, but I had entered the number without thinking.
I've never even heard of magic numbers before.
It's smarter to use ".length" to express the number in the array instead of writing it directly, and it won't cause bugs when the contents of the array change.
I learned a lot.